resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolverBuilder.java

```diff
@@ -25,6 +25,7 @@
 import io.netty.resolver.ResolvedAddressTypes;
 import io.netty.util.concurrent.Future;
 
+import java.net.InetSocketAddress;
```

Comment:
```suggestion
import java.net.SocketAddress;
```

Reply:
Sorry, I don't know I can commit the suggestion, I Fixed manually